src/pages/index.tsx

```diff
@@ -61,14 +61,14 @@ const HomePage: NextPage<
                 Fortress provides composable financial products for passive DeFi
                 investors.
               </p>
-              <ButtonLink
-                className="mt-6 px-8 lg:mt-8"
-                href={`${appUrl}/yield`}
-                size="large"
-                external
-              >
-                Launch App
-              </ButtonLink>
+              <a href={`${appUrl}/yield`} target="_self">
```

Comment:
I thought you would apply these hover classes to the button? We should use the components sir. 

All you have to do is add the hover classes to `buttonClasses`. 

If you're not going to use the component you have to at least use `next/link` instead of a raw anchor tag. Otherwise we lose benefits like preloading.